packages/mui-codemod/src/deprecations/backdrop-props/backdrop-props.test.js

```diff
@@ -0,0 +1,29 @@
+import path from 'path';
+import { expect } from 'chai';
+import { jscodeshift } from '../../../testUtils';
+import transform from './backdrop-props';
+import readFile from '../../util/readFile';
+
+function read(fileName) {
+  return readFile(path.join(__dirname, fileName));
+}
+
+describe('@mui/codemod', () => {
+  describe('deprecations', () => {
+    describe('backdrop-props', () => {
+      it('transforms props as needed', () => {
+        const actual = transform({ source: read('./test-cases/actual.js') }, { jscodeshift }, {});
+
+        const expected = read('./test-cases/expected.js');
+        expect(actual).to.equal(expected, 'The transformed version should be correct');
+      });
+
+      it('should be idempotent', () => {
+        const actual = transform({ source: read('./test-cases/expected.js') }, { jscodeshift }, {});
+
+        const expected = read('./test-cases/expected.js');
+        expect(actual).to.equal(expected, 'The transformed version should be correct');
+      });
+    });
+  });
```

Comment:
We should add theme tests, like this one:

- [test](https://github.com/mui/material-ui/pull/40771/files#diff-8f4e5d6bceb9d66d301f679a1fc4b0eceb1521045ef4996d188562f59a0a3a9eR29)
- [`theme.actual.js`](https://github.com/mui/material-ui/pull/40771/files#diff-c55d2074f8626bd471546b9956d1f2ee4d2cdbb43ed42a86b107e86ba6758db6)
- [`theme.expected.js`](https://github.com/mui/material-ui/pull/40771/files#diff-e2ad4861d6d895e34d400935f73269f7d73a9db9ef17f1a5105bb56ebc2ca290)